shell/components/fleet/FleetResources.vue

```diff
@@ -39,6 +39,8 @@ export default {
             state = perEntry.state;
           } else if ( tooMany ) {
             state = 'Unknown';
```

Comment:
Could this be an enum as well? since its not part of the change not a blocker. But its a quick change.